data/json/mutations/mutations.json

```diff
@@ -6104,8 +6104,60 @@
     "types": [ "HANDS" ],
     "prereqs": [ "PAWS" ],
     "cancels": [ "TALONS" ],
+    "changes_to": [ "PAWS_HUGE" ],
+    "category": [ "BEAST", "URSINE" ]
+  },
+  {
+    "type": "mutation",
+    "id": "PAWS_HUGE",
+    "name": { "str": "Crushing Paws" },
+    "points": 2,
+    "visibility": 6,
+    "ugliness": 3,
+    "mixed_effect": true,
+    "description": "Your broad paws are much larger now.  Manual dexterity is severely lower: permanent hand encumbrance of 40, serious problems crafting, and no gloves.  In return, though, you can swim better, and your hands can crush skulls.",
+    "craft_skill_bonus": [
+      [ "electronics", -6 ],
+      [ "tailor", -6 ],
+      [ "mechanics", -6 ],
+      [ "firstaid", -4 ],
+      [ "computer", -4 ],
+      [ "traps", -4 ],
+      [ "fabrication", -4 ],
+      [ "cooking", -4 ],
+      [ "survival", -4 ]
+    ],
+    "encumbrance_always": [ [ "hand_l", 40 ], [ "hand_r", 40 ] ],
+    "restricts_gear": [ "hand_l", "hand_r" ],
+    "bash_dmg_bonus": 15,
+    "flags": [ "UNARMED_BONUS" ],
+    "types": [ "HANDS" ],
+    "threshreq": [ "THRESH_URSINE" ],
+    "prereqs": [ "PAWS_LARGE" ],
+    "cancels": [ "TALONS" ],
     "category": [ "BEAST", "URSINE" ]
   },
+  {
+    "type": "mutation",
+    "id": "CRUSHING_FANGS",
+    "name": { "str": "Crushing Fangs" },
+    "points": 4,
+    "visibility": 3,
+    "ugliness": 3,
+    "description": "Your big teeth have grown in.  You can easily tear meat apart.",
+    "types": [ "TEETH" ],
+    "prereqs": [ "MUZZLE_BEAR" ],
+    "threshreq": [ "THRESH_URSINE" ],
+    "category": [ "URSINE" ],
+    "attacks": {
+      "attack_text_u": "You bite a chunk out of %s",
+      "attack_text_npc": "%1$s bites a chunk out of %2$s",
+      "blocker_mutations": [ "FANGS" ],
+      "body_part": "mouth",
+      "chance": 20,
+      "base_damage": [ { "damage_type": "stab", "amount": 10 }, { "damage_type": "bash", "amount": 10 } ]
```

Comment:
I'd be interested in how much damage this does as compared to FANGS on the same character.

Reply:
Same as fangs, 20. I thought about just adding fangs to it, maybe i should? i figured a bear would have some bash value along with their bite rather than puncture alone.